pkg/analysis/passes/binarypermissions/binarypermissions.go

```diff
@@ -20,7 +20,7 @@ var (
 var requiredPermissions = fs.FileMode.Perm(0755)
 
 var Analyzer = &analysis.Analyzer{
-	Name:     "archivename",
+	Name:     "binarypermissions",
```

Comment:
I missed this in my previous PR. It is not strictly part of these changes but fixes error reporting.